en/sensor/pmw3901.md

```diff
@@ -0,0 +1,176 @@
+Thone<br>![PMW3901 Thoneflow Notch](../../assets/hardware/sensors/pmw3901/thoneflow_3901u_notch.jpg) | Alientek (also has an arrow indicating the front!)<br>![PMW3901 Alientek Notch](../../assets/hardware/sensors/pmw3901/alientek_pmw3901_notch.jpg)
+
+
+## PX4 Configuration {#configuration}
+
+The PX4 configuration that is common to all boards:
+- [Optical Flow > EKF2](../sensor/optical_flow.md#ekf2) explains how to fuse optical flow data in the EKF2 estimator and set position offsets for the mounting position of the flow sensor.
+- [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT) sets the orientation of the flow sensor relative to vehicle heading.
+
+In addition:
+- The distance sensors should be enabled as described in their setup documentation (no flow-specific configuration is required).
+- Individual flow sensors are setup/configured as described in the sections below.
+
+
+## Bitcraze Flow breakout {#bitcraze_flow_breakout}
+
+The [Bitcraze Flow breakout](https://www.bitcraze.io/products/flow-breakout/) directly exposes the [SPI interface](#spi_wiring) from the PMW3901 module.
+
+The board also incorporates a distance sensor that can be [wired to the Pixhawk I2C port](#i2c_wiring).
+This distance sensor is the VL53L0x ToF sensor from STMicroelectronics.
+The range of the sensor is very limited (2 meters) and will be reduced when flying in the sunlight.
+We therefore highly recommend that you use an [external distance sensor](#external_rangefinder).
+
+![Bitcraze Flow Deck](../../assets/hardware/sensors/pmw3901/bitcraze-flow.jpg)
+
+
+### SPI Wiring {#spi_wiring}
+
+The PMW3901 driver was written such that the board can be directly plugged into the SPI port on the Pixhawk 4 using the chip select 1.
+When plugged into the SPI port of the Pixhawk 4, the Bitcraze flow module will be automatically detected at bootup.
+No parameters will have to configured other than the [orientation and position of the sensor](#orientation).
+
+The pinout mapping for the Pixhawk SPI port to Bitcraze Flow Board is shown below (the port mapping is the same for all Pixhawk FMU versions).
+
+Pixhawk SPI Port (from left to right) | Bitcraze flow board
+--- | ---
+1 (VCC) | VCC
+2 (SCK) | CLK
+3 (MISO) | MISO
+4 (MOSI) | MOSI
+5 (CS1) | Do not connect
+6 (CS2) | CS
+7 (GND) | GND
+
+In order to connect the bitcraze flow board to the Pixhawk you will need to solder the wires of the Pixhawk SPI cable to the flow board.
+An SPI cable has 7 wires, from which you need to connect 6 to the flow board.
+
+The following diagram shows how to wire the sensor to a Pixhawk 4.
+
+![Bitcraze PH4 Pinout](../../assets/hardware/sensors/pmw3901/ph4-bitcraze-flow-pinout.png)
+
+
+> **Tip** The SPI support for this driver can be extended to other boards by modifying the `SPI_BUS` and the `SPI_DEV` in the [PMW3901 driver](https://github.com/PX4/Firmware/tree/{{ book.px4_version}}/src/drivers/optical_flow/pmw3901).
```

Comment:
Meanwhile this has changed, and is not required anymore. With `SENS_EN_PMW3901`, the driver will automatically be started on the 1. external SPI bus, 1. chip-select signal.

Reply:
Thanks @bkueng 
I just deleted the section: https://github.com/PX4/px4_user_guide/commit/41c10cf75fef0470f36f94563fead726a77548ed